app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -65,4 +91,10 @@ INSTALL()
 		LUA_PACKAGE=lua
 	mv $docDir/wordgrinder/README.wg $docDir
 	rmdir $docDir/wordgrinder
+if [ -z "$secondaryArchSuffix" ]; then
+	packageEntries dictionaries \
+		$dataDir
+fi
+	install -d $dataDir/wordgrinder/dictionaries
+	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary
```

Comment:
Oops, the last two lines (which install the dictionaries) are not in the right place.
This is what we should have:
```
INSTALL()
{
	make install PREFIX=$prefix BINDIR=$commandBinDir DOCDIR=$docDir \
		MANDIR=$manDir HOME="`finddir B_USER_DIRECTORY`" \
		LUA_PACKAGE=lua
	mv $docDir/wordgrinder/README.wg $docDir
	rmdir $docDir/wordgrinder

	install -d $dataDir/wordgrinder/dictionaries
	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary
if [ -z "$secondaryArchSuffix" ]; then
	packageEntries dictionaries \
		$dataDir
fi
}
```